### PR TITLE
add message client and unit test.

### PR DESCRIPTION
**Pull request by vwxyzh**

### CODE REVIEW

File: src/Tests/TestCommon/TaskExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.SignalRBench.Tests
+{
+    public static class TaskExtensions
+    {
+        public static async Task WithTimeout(this Task task, TimeSpan? timeout = default)
```
Comment (wanlwanl) [inline src/Tests/TestCommon/TaskExtensions.cs:11]: How about `OrTimeout`, keep the same with server SDK?

File: src/Tests/MessagesTest/MessagesTest.csproj [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    <RootNamespace>Microsoft.Azure.SignalRBench.Tests.MessagesTest</RootNamespace>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0" />
+    <PackageReference Include="xunit" Version="2.4.0" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.0" />
+    <PackageReference Include="coverlet.collector" Version="1.2.0" />
```
Comment (wanlwanl) [inline src/Tests/MessagesTest/MessagesTest.csproj:14]: How about using `dependencies.prop` to store versions?

File: src/Tests/TestCommon/Requirements.cs [hunk 3]
```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Xunit;
+
+namespace Microsoft.Azure.SignalRBench.Tests
+{
+    public static class Requirements
```
Comment (wanlwanl) [inline src/Tests/TestCommon/Requirements.cs:8]: How about use `Atttibute` like [`SkipIfConnectionStringNotPresentAttribute`](https://github.com/Azure/azure-signalr/blob/d9bea3cce0df358f6b35c9f5d88969c1dde6f25e/test/Microsoft.Azure.SignalR.Tests.Common/SkipIfConnectionStringNotPresentAttribute.cs). It looks clearer

Comment (vwxyzh) [reply src/Tests/TestCommon/Requirements.cs:8]: I cannot find that package in nuget.org

File: src/Tests/MessagesTest/MessagesTest.csproj (hunk 2 above)
Comment (vwxyzh) [reply src/Tests/MessagesTest/MessagesTest.csproj:14]: good idea, but keep it todo.

**Pull request merged**